docs/client_reference.rst

```diff
@@ -995,6 +995,7 @@ Response object
       :param callable loads: :func:`callable` used for loading *JSON*
                              data, :func:`json.loads` by default.
 
-      :return dict: *BODY* as *JSON* data.
+      :rtype: dict, list, str, int, float, bool or None
```

Comment:
should be 
    :class:`dict`
etc.
Also please note the method returns `None` for empty body.